rl:
  robot:
    compute_power: 1000e-3  # Watt
    transmit_power: 100e-3  # Watt
  reward:
    w_latency: 0.33
    w_energy: 0.33
    w_model: 0.33